charts/spire/charts/spire-agent/values.yaml

```diff
@@ -106,6 +106,20 @@ waitForIt:
     tag: latest-20230517
   resources: {}
 
+fsGroupFix:
```

Comment:
Can you add a comment above here describing what this section is for?

Reply:
done